### PR TITLE
Show default values for debug-assertions & debug-assertions-std

### PR DESCRIPTION
**Pull request by tmiasko**

### CODE REVIEW

File: config.toml.example [hunk 1]
```diff
@@ -312,11 +312,11 @@
 
 # Whether or not debug assertions are enabled for the compiler and standard
 # library.
-#debug-assertions = false
+#debug-assertions = debug
```
Comment (nikomatsakis) [inline config.toml.example:315]: is this legal syntax if you uncomment it?

Comment (tmiasko) [reply config.toml.example:315]: It is invalid when uncommented. But I think it is valuable to indicate how default value is obtained, and similar approach is already used for other options.

**Comment by nikomatsakis** [comment]
Hmm, ok. I agree it's useful though I'm surprised we would permit syntax that can't be uncommented. But if we follow the same approach elsewhere, seems ok.

**Pull request merged**